frontend/src/components/CellView.tsx

```
@@ -12,21 +14,46 @@ interface CellViewProps {
+    p2Decided: boolean;
+    p2Wants: boolean;
+  } | null>(null);
+
   if (!userAddress) return null;
 
   const isPlayer1 = cell.player1.toLowerCase() === userAddress.toLowerCase();
   const isPlayer2 = cell.player2.toLowerCase() === userAddress.toLowerCase();
   const isParticipant = isPlayer1 || isPlayer2;
 
+  // Poll continuation status when round is complete and cell is not complete
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)